### ISSUE TITLE
Errors when PulseAudio is restarted

### ISSUE THREAD

**Issue by lahwaacz** [natsukagami/mpd-mpris#19]
Whenever I restart PulseAudio, mpd-mpris starts spamming the logs with several messages per second like this:
```
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 Volume changed to 0
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 command 'setvol' failed: problems setting volume
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 Volume changed to 0
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 command 'setvol' failed: problems setting volume
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 Volume changed to 0
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 command 'setvol' failed: problems setting volume
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 Volume changed to 0
Nov 20 22:56:13 jlknb mpd-mpris[947]: 2020/11/20 22:56:13 command 'setvol' failed: problems setting volume
...
```

I'm using version 0.3.0.

**Comment by natsukagami** [comment]
This is due to `mpd` (and `mpd-mpris`) not handling volume correctly when there's no audio output I think. `mpd` keeps reporting that `volume = 0` and when `mpd-mpris` sees the newly set volume, it tries to update the MPRIS interface value, which triggers itself to write the new volume value to `mpd`. However `mpd` refuses to update the volume.

Manually I would just stop/play on `mpd` again or switch to a different song (I don't think this will happen when `mpd` is stopped).
Perhaps I will implement some sort of back-off counter for the periodical updater for `mpd-mpris`, so it wouldn't spam the log with failed updates.

**Comment by lahwaacz** [comment]
Well, `mpd` playback is paused when PulseAudio is restarted, so I need to resume it again which resets the volume problem in `mpd-mpris`. But there is really no reason for repeated output, there should be at most one error message written to the log in such cases.

**Comment by natsukagami** [comment]
This should be resolved by #34 as well, since we no longer poll for volume from mpd.
